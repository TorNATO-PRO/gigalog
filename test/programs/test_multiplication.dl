product(X, Y, Z) :- Z = X * Y.

product(2, 3, 6).
product(4, 5, 20).
product(7, 8, 56).
