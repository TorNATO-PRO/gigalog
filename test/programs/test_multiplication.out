product(2,3,6)
product(4,5,20)
product(7,8,56)
